dts/bindings/clock/st,stm32-rcc.yaml

```diff
@@ -11,6 +47,44 @@ properties:
     "#clock-cells":
       const: 2
 
+    clock-frequency:
+      required: false
+      type: int
+      description: |
+        default frequency in Hz for clock output
+
+    ahb-prescaler:
+      type: int
+      required: false
+      enum:
+        - 1
```

Comment:
This is a bit weird; do you really need to have this property now in this way?

Reply:
I agree this is a bit weird but I wanted this property to be present for 2 reasons:
- backward compatibility with existing scheme
- this is a typical STM32 configuration piece. We have a tool to help users configure clocks and this is part of the tool output. So, despite the current constraint on this field in Zephyr, I don't want to hide it in any way in order not to disturb users and to let them inform the earliest of this fact.

Aside the weirdness, is there any concern using 'enum' this way ?

Reply:
Makes sense, thanks